packages/ui/src/components/Form/styles.css.ts

```diff
@@ -83,6 +83,7 @@ export const inputVariants = recipe({
 		},
 	},
 	defaultVariants: {
+		size: 'small',
```

Comment:
Noticed the height of form inputs was slightly off from the mocks. All form inputs were supposed to be 28px tall, which this enforces.